contrib/hawq-docker/Makefile

```diff
@@ -39,6 +41,17 @@ all:
 	@echo ""
 	@echo "    To build images locally:                       make build"
 	@echo "    To pull latest images:                         make pull"
+	@echo ""
+	@echo "    To build Hawq runtime:                         make hawq"
```

Comment:
'make hawq' better to be 'make build-hawq', 'make build' better to be 'make build-image'. If changed, please update  related code and the README as well.

Reply:
Done